spec/models/author_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Author, type: :model do
+  context 'database associations' do
+    it { is_expected.to have_many(:items) }
+  end
+
+  context 'model validations' do
+    it { is_expected.to validate_presence_of(:name) }
+
+    it { is_expected.to validate_presence_of(:service) }
+    it { is_expected.to validate_inclusion_of(:service).in_array(ServiceKind.list) }
+  end
+
+  context 'model attributes' do
+    it { is_expected.to have_db_column(:id).of_type(:integer).with_options(null: false) }
+
+    it { is_expected.to have_db_column(:service).of_type(:integer)
+                            .with_options(null: false) }
```

Comment:
Expression at 19, 56 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 24) spaces for indenting an expression spanning multiple lines.